### PR TITLE
Replaced file copying code with an existing utility function.

### PR DESCRIPTION
**Pull request by tomob**
Signed-off-by: Tomasz Baranski <tbaransk@redhat.com>

**What this PR does / why we need it**:

Technical debt. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/importer/transport.go [hunk 1]
```diff
@@ -162,15 +163,9 @@ func processLayer(ctx context.Context,
 				return false, errors.Wrap(err, "Error creating output file's directory")
 			}
 
-			dstFile, err := os.Create(filepath.Join(destDir, hdr.Name))
-			if err != nil {
-				klog.Errorf("Error creating output file: %v", err)
-				return false, errors.Wrap(err, "Error creating output file")
-			}
-
-			if err := copyFile(tarReader, dstFile); err != nil {
```
Comment (maya-r) [inline pkg/importer/transport.go:171]: you can probably delete the copyFile function now

**Comment by maya-r** [comment]
It looks like StreamDataToFile adds a sync before closing the file.
This might be a good thing to have in release branches (for various network based filesystems).
/lgtm

File: pkg/importer/transport.go (hunk 1 above)
Comment (tomob) [reply pkg/importer/transport.go:171]: Of course!

**Review by awels — APPROVED** [review]
/lgtm
/approve

**Pull request merged**